tests/roles/pcp_cleanup/defaults/main.yaml

```diff
@@ -8,3 +8,4 @@ reset_crc_storage: false
 # Whether 'make standalone_revert' should be run before
 # running the test suite.
 standalone_revert_enabled: false
+delay: 5
```

Comment:
In addition to breaking the lint, this is unused, i think it can be dropped. Other than that LGTM